### PR TITLE
Material Palette + HTML Markup Page

### PR DESCRIPTION
**Pull request by mavieth**
Added a few typography and a color palette options as well as an HTML markup page.

### CODE REVIEW

File: src/scss/styles.scss [hunk 1]
```diff
@@ -1,20 +1,51 @@
...
-$action-color: #477DCA !default;
-$dark-gray: #333 !default;
-$light-gray: #DDD !default;
+$base-heading-top-spacing: 1em !default;
+
+// Media Queries
 $medium-screen: em(640) !default;
-$base-font-color: $dark-gray !default;
+
+// Color Palette
```
Comment (zackhall) [inline src/scss/styles.scss:17]: I do like the idea of bringing a Material Palette in here.

File: src/scss/styles.scss [hunk 2]
```diff
@@ -1,20 +1,51 @@
 @import "bourbon";
 @import "neat";
 
+@import url(http://fonts.googleapis.com/css?family=Raleway:400,600,700,300);
+@import url(http://fonts.googleapis.com/css?family=Roboto:400,600,700,300);
+
```
Comment (zackhall) [inline src/scss/styles.scss:6]: In the master branch I'd like to keep the number of fonts down. I think these belong in each users' defined style for their site.

File: gulpfile.js [hunk 3]
```diff
@@ -89,6 +89,7 @@ gulp.task('serve', ['build'], function() {
   });
 
   gulp.watch('src/**/*.html', ['build', 'reload']);
+  gulp.watch('src/pages/**/*.md', ['pages', 'reload']);
```
Comment (zackhall) [inline gulpfile.js:92]: Good catch

File: src/scss/styles.scss (hunk 2 above)
Comment (mavieth) [inline src/scss/styles.scss:6]: I agree with. It was mainly just testing the reload after changing fonts + saving.

File: src/pages/markup.html [hunk 4]
```diff
@@ -0,0 +1,272 @@
+{% extends '_layout.html' %}
+
+{% block title %} {{ super() }} | Markup {% endblock %}
+
+{% block content %}
+
+<img src="http://placehold.it/1000x250" height="250">
+
+<h1>Hello</h1>
```
Comment (mavieth) [inline src/pages/markup.html:9]: This markup file is kind of a mess - mainly just seeing how different elements would look. I also wasn't sure if you intended on showing styles through the markdown pages (style ramp post).

File: src/scss/styles.scss (hunk 2 above)
Comment (zackhall) [inline src/scss/styles.scss:6]: Gotcha, gotcha. I can't really pull this as is, but if you want to clean up the .scss file to make the minimal changes you'd like to see in the master repo and add another commit I'd be happy to pull it.

File: src/pages/markup.html (hunk 4 above)
Comment (zackhall) [inline src/pages/markup.html:9]: Yeah, I'd like to expand that page to show the styles. Or have a singular page to show all the styles. But I like that this one is much more complete.

File: src/scss/styles.scss (hunk 2 above)
Comment (mavieth) [inline src/scss/styles.scss:6]: Cool. I am like 87% sure it was my first pull request haha. I have seen
some projects that use the material palette and then $brand-variable naming
convention that seem to organize things well.

On Fri, Mar 4, 2016 at 1:45 AM, Zack Hall notifications@github.com wrote:

> In src/scss/styles.scss
> https://github.com/zackhall/chug/pull/2#discussion_r54995502:
> 
> > @@ -1,20 +1,51 @@
> >  @import "bourbon";
> >  @import "neat";
> > 
> > +@import url(http://fonts.googleapis.com/css?family=Raleway:400,600,700,300);
> > +@import url(http://fonts.googleapis.com/css?family=Roboto:400,600,700,300);
> > +
> 
> Gotcha, gotcha. I can't really pull this as is, but if you want to clean
> up the .scss file to make the minimal changes you'd like to see in the
> master repo and add another commit I'd be happy to pull it.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/zackhall/chug/pull/2/files#r54995502.

## 

Michael Vieth

File: src/scss/styles.scss [hunk 5]
```diff
@@ -1,20 +1,51 @@
...
 $sans-serif: 'Merriweather Sans', sans-serif;
 
+// Brand Styles
+$brand-primary-type: $serif-2;
+$brand-primary-type-sans-serif: $sans-serif;
+$brand-type-headings: $serif-3;
+
 
-//* General Site Styles
+/* ==========================================================================
```
Comment (mavieth) [inline src/scss/styles.scss:46]: Not sure what your preference for comment styling is but I use the Sublime package [CSS Comments](https://github.com/brenopolanski/css-comments-sublime-snippets).

**Comment by mavieth** [comment]
I am going to work on trying to integrate something similar to WP with forms to create posts/pages.

**Pull request closed without merging**